### ISSUE TITLE
i can't write on the name and url fields on done button

### ISSUE THREAD

**Issue by SrStick** [sonnyp/Tangram#51]
i'm using Linux Mint 19.2

**Comment by sonnyp** [comment]
Can you share a screencast or something similar?

**Comment by sonnyp** [comment]
Are you using the flatpak from flathub?

What is your DE? Cinnamon? Mate? XFCE?

**Comment by SrStick** [comment]
![up](https://user-images.githubusercontent.com/28923770/64896987-2358d080-d658-11e9-9b92-9f9cde587406.png)

Yes, I'm using Flatpak from Flathub and my DE is XFCE.

**Comment by sonnyp** [comment]
Thanks for the report.

I can reproduce in a VM, everything entered into the fields gets immediately remove because of the GSettings binding not working properly. I'm not sure what it is exactly but upgrading Flatpak fixed the issue.

```
sudo add-apt-repository ppa:alexlarsson/flatpak
sudo apt update
sudo apt upgrade
flatpak run re.sonny.Tangram
```

An other issue on Mint/XFCE is that the desktop (not application) icon is incorrect.

I'll try to debug this soon-ish.

**Comment by sonnyp** [comment]
Same issue on Ubuntu 18.04.

![Screenshot from 2019-09-14 13-25-52](https://user-images.githubusercontent.com/19673/64907478-3bc9f900-d6f3-11e9-88e6-cd8231afc09e.png)

It might not be a big issue there though because Flathub recommends to install flatpak from the PAA on Ubuntu < 18.10 https://flatpak.org/setup/Ubuntu/

However for Mint, which the latest version is based on Ubuntu 18.04, the setup instructions do not recommend the PPA. https://flatpak.org/setup/Linux%20Mint/

@SrStick Would you like to update the instructions for mint to recommend to use the PPA, much like instructions for Ubuntu? If so you can edit https://github.com/flatpak/flatpak.github.io/blob/source/data/distro.yml and send a PR.

**Comment by sonnyp** [comment]
Turns out updating to Gnome SDK 3.34 fixes the issue, preparing a new release.

**Comment by sonnyp** [comment]
released 1.1.0 https://github.com/sonnyp/Tangram/releases/tag/v1.1.0

`flatpak update re.sonny.Tangram` please confirm the issue is fixed for you and close this

**Comment by SrStick** [comment]
I did the update and the bug was fixed
